impeller/renderer/backend/gles/render_pass_gles.cc

```diff
@@ -501,9 +508,11 @@ bool RenderPassGLES::EncodeCommands(
   }
 
   return reactor_->AddOperation([pass_data, transients_allocator,
-                                 commands = commands_](const auto& reactor) {
+                                 commands = commands_,
+                                 target_size = GetRenderTargetSize()](
```

Comment:
Isn't this the same as `pass_data->color_attachment`s size? I don't think there is a need to add another capture.

Reply:
Ah, good point. Done.